index.html

```diff
@@ -42,5 +43,185 @@ <h2 class="Supportingtext Title2">I'm a Software Developer</h2>
       </ul>
     </div>
   </section>
+
+<!-- Section: Recent Works -->
+<section class="works_main">
+
+  <div class="works_title">
+    <h3 class="Title-work-main Title2_dark">My recent Works</h3>
+    <span class="line_div"></span>
+  </div>
+
+  <div class="main_card_work">
+    <div class="main_card">
+      <div class="cnt_img_main_card">
+        <img class="work_img" src="/images/card1_img.png" alt="Project image">
+      </div>
+      <div class="cnt_info_main_card">
+        <h3 class="main_work Title2_dark">Multi-Post Stories</h3>
+        <p class="Simpletext">A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="list_feat list_card_1st">
+          <li class="li_btn">CSS</li>
+          <li class="li_btn">HTML</li>
+          <li class="li_btn">BOOTSTRAP</li>
+          <li class="li_btn">RUBY</li>
+        </ul>
+        <a class="project_btn_all project_btn_1" href="#">See Project</a>
+      </div>
+    </div>
+  </div>
+
+  <div class="grid_works">   
+    <div class="card_cnt work-1">
+      <div class="info_card">
+        <div class="cnt_info_card">
+          <h4 class="Title1_card">Pet Shopping Store</h4>
+          <p class="Title2_card">Here you can find the best productos for your pet</p>
+          <ul class="list_feat list_card_gral">
+            <li class="li_btn_2">HTML</li>
+            <li class="li_btn_2">Bootstrap</li>
+            <li class="li_btn_2">Ruby</li>
+          </ul>
+        </div> 
+        <a href="#" class="project_btn_all project_btn_2">See Project</a>
+      </div>
+    </div>
```

Comment:
Please consider matching your design with the template. Currently, you have a different set of images compared to the mobile version of the design. Also, there are different styles regarding the heading and other text shown.

## Your Design:

![image](https://user-images.githubusercontent.com/94223367/227763547-437d89eb-b59d-4a06-b5c9-55a54d2dc835.png)


## Figma Design:

![image](https://user-images.githubusercontent.com/94223367/227763579-4c4a2f1d-f557-4c90-8ab8-fa52e5165a0d.png)


## REMEMBER:-

This problem exists in all of the project cards so consider reviewing and rectifying it in all of them.

Reply:
Done. Thank you @ABDULALI3468